azurerm/internal/services/network/virtual_network_gateway_resource_test.go

```diff
@@ -179,10 +179,6 @@ func TestAccVirtualNetworkGateway_vpnClientConfigAzureAdAuth(t *testing.T) {
 			Config: r.vpnClientConfigAzureAdAuth(data),
 			Check: resource.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
-				check.That(data.ResourceName).Key("vpn_client_configuration.0.aad_tenant").HasValue(fmt.Sprintf("https://login.microsoftonline.com/%s/", data.RandomString)),
-				check.That(data.ResourceName).Key("vpn_client_configuration.0.aad_audience").HasValue("41b23e61-6c1e-4545-b367-cd054e0ed4b4"),
-				check.That(data.ResourceName).Key("vpn_client_configuration.0.aad_issuer").HasValue(fmt.Sprintf("https://sts.windows.net/%s/", data.RandomString)),
-				check.That(data.ResourceName).Key("vpn_client_configuration.0.vpn_client_protocols.#").HasValue("1"),
```

Comment:
If we are removing these checks, we should add an import step instead. Or we could fix the checks by specifying the correct tenant ID etc

Reply:
@manicminer Sorry about this, I forgot to check this. The test passed after adding the ImportStep:

```bash
💢 TF_ACC=1 go test -v -timeout=3h ./azurerm/internal/services/network -run="TestAccVirtualNetworkGateway_vpnClientConfigAzureAdAuth"
2021/05/18 09:19:25 [DEBUG] not using binary driver name, it's no longer needed
2021/05/18 09:19:25 [DEBUG] not using binary driver name, it's no longer needed
=== RUN   TestAccVirtualNetworkGateway_vpnClientConfigAzureAdAuth
=== PAUSE TestAccVirtualNetworkGateway_vpnClientConfigAzureAdAuth
=== CONT  TestAccVirtualNetworkGateway_vpnClientConfigAzureAdAuth
--- PASS: TestAccVirtualNetworkGateway_vpnClientConfigAzureAdAuth (1677.17s)
PASS
ok      github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network     1677.380s
```